src/chapters/installing.xml

```diff
@@ -452,6 +461,31 @@
                                 </informaltable>
                             </entry>
                         </row>
+                        <row>
+                            <entry><link xl:href="https://github.com/seanmiddleditch/libtelnet"
+                                    >libwebsockets</link></entry>
```

Comment:
Is this for `libtelnet` or `libwebsockets`?  Link is to `libtelnet`, but link text is for `libwebsockets`??

Reply:
Whoops. Copypasta. It's supposed to be for libwebsockets.